framework/lib/components/combo-picker/combo-picker-field.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import { useFormContext } from '../../hooks'
+import { ComboPickerFieldProps, ComboPickerValue } from './types'
+import { ComboPicker } from './combo-picker'
+import { Field } from '../form'
+
+/**
+ * The <ComboPicker /> component wrapped in a <Field />
+ * meant to be used only inside <Form />
+ * @see ComboPicker
+ * @see {@link https://northlight.dev/reference/combo-picker}
+ *
+ * @example (Example)
+ * ##Fill in your information:
+ * (?
+ * <Form initialValues={{}}>
+ * <ComboPickerField
+ *         label="Money spent"
+ *         name="spent"
+ *         value={
+ *           {
+ *             input: 195,
+ *             option: {
+ *               label: 'SEK',
+ *               value: 'sek',
+ *             },
+ *          }
+ *         }
+ *         options={ [
+ *           { label: 'USD', value: 'usd' },
+ *           { label: 'EUR', value: 'eur' },
+ *           { label: 'SEK', value: 'sek' },
+ *        ] }
+ *      />
+ * </Form>
+ *
+ * ?)
+ * <br />
+ */
+export const ComboPickerField = ({
+  name,
+  isRequired,
+  direction = 'column',
+  label,
+  validate,
+  onChange: onChangeCallback,
+  ...rest
+}: ComboPickerFieldProps) => {
+  const { setValue } = useFormContext()
+
+  const handleChange = (comboPickerValue: ComboPickerValue) => {
+    setValue(name, comboPickerValue)
```

Comment:
I added it because when selecting an option from the select list, I also want to trigger the onChange callback, sending both the inputValue and the selected option, and I need access to the value for it!

Reply:
Yeah you can still call the onChangeCallback and keep the handleChange callback, but line 52 that calls setValue is redundant